docs/development_guide/repo_structure.md

```diff
@@ -0,0 +1,154 @@
+# Repository Structure
+
+```angular2html
+├── cloudbuild.yaml
+├── LICENSE
+├── README.md
+├── mkdocs.yml
+├── requirements.txt
+├── tox.ini
+├── VERSION
+├── docker
+│   └── ...
+├── docs
+│   └── ...
+├── resources
+│   └── ...
+├── scripts
+│   └── ...
+├── terra
+│   └── ...
+├── test
+│   └── ...
+└── wdl
+    └── ...
+```
+
+
+The repository file and directory is as follows:
+
+* LICENSE: The license for the repository.
+* README.md: This document, which provides an overview of the repository.
+* VERSION: The version number of the repository.
+* cloudbuild.yaml: A Cloud Build configuration file that defines how the repository is built.
+* docker: A directory containing Dockerfiles for building the pipelines.
+* docs: A directory containing documentation for the pipelines.
+* requirements.txt: A file listing the Python dependencies for the pipelines.
+* resources: A directory containing resources used by the pipelines.
+* mkdocs.yml: A configuration file for the mkdocs documentation generator.
+* scripts: A directory containing scripts used to build and run the pipelines.
+* test: A directory containing unit tests for the pipelines.
+* tox.ini: A configuration file for the tox test runner.
+* wdl: A directory containing WDL files for the pipelines.
+
+## WDL Directory Structure:
+
+```angular2html
+└── wdl
+    └── pipelines
+    │  └── ...
+    └── structs
+    │  └── ...
+    └── tasks
+         └── ...
+```
+The WDL directory is further divided into subdirectories. The subdirectories are as follows:
+* tasks: A directory containing WDL tasks for the pipelines.
+* pipelines: A directory containing WDL workflows for the pipelines.
+* structs: A directory containing WDL structs for the pipelines.
+
+### Tasks Directory Structure:  
+The task directory has an additional subdirectory to organize wdl tasks by analysis type. The subdirectories are as follows:  
+
+```angular2html
+└── wdl
+    └── tasks
+    │  └── alignment
+    │  │  └── ...
+    │  └── annotation
+    │  │  └── ...
+    │  └── assembly
+    │  │  └── ...
+    │  └── epigenomics
+    │  │  └── ...
+    │  └── preprocessing
+    │  │  └── ...
+    │  └── qc
+    │  │  └── ...
+    │  └── transcriptomics
+    │  │  └── ...
+    │  └── utility
+    │  │  └── ...
+    │  └── variantcalling
+    │  │  └── ...
+    │  └── visualization
+    │  │  └── ...
+```
+
+* alignment: A directory containing WDL tasks for aligning reads to a reference genome.
```

Comment:
This can probably be revised a little to avoid repeating the phrase "A directory containing WDL tasks for " so many times. It would be easier to read that way.

Reply:
I'm beginning to think a description isn't needed for these folders since the name is self explanatory